assets/sass/components/user-input/googlesitekit-user-input-questions.scss

```diff
@@ -115,4 +115,10 @@
 			width: 24px;
 		}
 	}
+
+	.googlesitekit-user-input__question--progress {
+		margin: 0 auto;
+		max-width: calc(100% - #{$grid-gap-phone * 2});
+		max-width: calc(100% - #{$grid-gap-desktop * 2});
```

Comment:
@Hazlitte this will override max-width for phones defined one line above. We need to wrap this line with a media query like suggested in the IB:

>  .googlesitekit-user-input__question--progress {
> 	margin: 0 auto;
> 	max-width: calc(100% - #{$grid-gap-phone * 2});
>
> 	@media (min-width: $bp-tablet) {
> 		max-width: calc(100% - #{$grid-gap-desktop * 2});
> 	}
> }

Reply:
👍   Hi @eugene-manuilov,  I did try the suggested media query but it breaks the alignment at the 600px break point.  Line 122 seemed to work better when I tested it.  I forgot to remove line 121, my bad!

Reply:
I think you should use `$bp-desktop` instead of `$bp-tablet` and then it should work fine:

```suggestion
		max-width: calc(100% - #{$grid-gap-phone * 2});
		
		@media (min-width: $bp-desktop) {
			max-width: calc(100% - #{$grid-gap-desktop * 2});
		}
```

Can you try this?

Reply:
@eugene-manuilov I just tested that and it works down to the break point of 960px.  Below that the progress bar fails to align with the content below as per the AC.

![Screenshot 2021-03-31 at 15 35 30](https://user-images.githubusercontent.com/5174233/113162224-01716f80-9237-11eb-8326-401f25477424.png)

Reply:
@Hazlitte let's update styles for the content below to have the same gaps on screens shorter than 960px

Reply:
👍